server/chat-commands/moderation.ts

```diff
@@ -869,7 +869,7 @@ export const commands: Chat.ChatCommands = {
 
 		if (!targetUser && !Punishments.search(userid).length && !force) {
 			return this.errorReply(
-				`User '${targetUsername}' not found. Use \`\`/forcelock\`\` if you need to to lock them anyway.`
+				`User '${targetUsername}' not found. Use \`\`/force${month ? 'month' : week ? 'week' : ''}lock\`\` if you need to to lock them anyway.`
```

Comment:
```suggestion
				`User '${targetUsername}' not found. Use \`\`/force${month ? 'month' : (week ? 'week' : '')}lock\`\` if you need to to lock them anyway.`
```
This makes it more readable.

Reply:
I always prefered chaining ternaries without the parentheses, but I suppose that's fine